layout/header/constants.js

```diff
@@ -16,19 +16,19 @@ export const APP_HEADER_ITEMS = [
     ]
   },
   {
-    id: 'topics',
-    label: 'Topics',
-    route: 'topics',
-    pathnames: ['/app/topics', '/app/topics-detail'],
+    id: 'dashboards',
+    label: 'Dashboards',
+    route: 'dashboards',
+    pathnames: ['/app/About', '/app/Partners'],
```

Comment:
@dancer2090 I don't understand this change, why did you set `About` and `Partners` as pathnames?